gettingstarted/index.md

```diff
@@ -2,7 +2,462 @@
 layout: page
 ---
 
-Getting Started
----------------
+# Getting Started #
 
-Coming soon...
+This getting started / quickstart guide will help get your first Naptime resource up and running.
+
+## An Example Resource: Users
+
+Let's walk through an example of exposing a simple User resource. The full example resource can
+be found at: <https://github.com/coursera/naptime/examples/TODO>.
+
+A user has an id, a name, an email, and an optional description of self (long
+blob of text).
+
+<small>Note: For clarity, imports / sections of code may be snipped.</small>
+
+### Setting Up
+
+#### Installing Naptime
```

Comment:
Consider linking to Courier setup instructions since that requires a plugin (right?) and we're recommending it.